app/models/kpi.rb

```diff
@@ -0,0 +1,366 @@
+class Kpi < CouchRest::Model::Base
```

Comment:
Wouldn't a better model name be "KeyPerformanceIndicator"?   I had to ask what "KPI" meant.